package syncro.pb;

option java_package = "uk.me.grambo.syncro.pb";
option optimize_for = LITE_RUNTIME;

message FolderListRequest {
	optional string search_string = 1;
	optional bool writable = 2;
	optional bool readable = 3;
}

message FolderInfo {
	required int32 folder_id = 1;
	optional string folder_name = 2;
	optional bool can_write = 3;
	optional bool can_read = 4;
};

message FolderList {
	repeated FolderInfo folders = 1;
};